templates/default/css/template.css

```diff
@@ -89,12 +96,18 @@ padding-top:1em;padding-bottom:1em;margin-bottom:2em;
 .answer-container {
   border-width: 0px 1px 0px 1px;
   border-style: solid;
-  padding-top: 3em;
-  padding-bottom:1.5em;
+  //padding-top: 3em;
+  //padding-bottom:1.5em;
```

Comment:
You put in margin, while it should have been padding. Look at box around answers..